Mathlib/Computability/AkraBazzi/AkraBazzi.lean

```diff
@@ -585,8 +585,8 @@ lemma asympBound_def' {n : ℕ} :
   simp [asympBound_def, sumTransform, mul_add, mul_one, Finset.sum_Ico_eq_sum_range]
 
 lemma asympBound_pos (n : ℕ) (hn : 0 < n) : 0 < asympBound g a b n := by
-  calc 0 < (n : ℝ) ^ p a b * (1 + 0)   := by aesop (add safe Real.rpow_pos_of_pos)
-       _ ≤ asympBound g a b n    := by
+  calc 0 < (n : ℝ) ^ p a b * (1 + 0) := by aesop (add safe Real.rpow_pos_of_pos)
```

Comment:
What is the alignment convention for proofs in `calc`? We seem to have both here.

Reply:
adomani/linter_stain_active